antlr-kotlin-runtime/build.gradle.kts

```diff
@@ -51,24 +51,28 @@ kotlin {
   }
 }
 
+mavenPublishing {
```

Comment:
Do we have a strong reason to change the publishing strategy? It was so difficult to make it work that I would be reluctant to change it, unless it is strictly needed

Reply:
Sorry, I noticed you just moved it, you did not add it

Reply:
To clarify, the publishing strategy hasn't changed: the difference is you can now run two separate publications:

- `publishAllPublicationsToMavenCentralRepository` for Maven Central
- `publishAllPublicationsToPrivateNexusRepository` for whatever private repository you've configured